### ISSUE TITLE
please make archunit.org website public (and not redirect to github repo)

### ISSUE THREAD

**Issue by gernotstarke** [TNG/ArchUnit#55]
http://archunit.org still redirects me to the github repository...

most likely NOT what was intended when creating the docs directory with all the nice jekyll stuff in it...

**Comment by codecholeric** [comment]
This was what I had intended, while the state was "almost done"(TM) :wink: 
Just wanted to make sure, that everything relevant from the README.md found its way into the new shiny user guide.
But yes, if you check the link now, you should be satisfied...
In my opinion this solves:
- this issue
- #30 -> I've created a new repo to house released docs + examples: https://github.com/TNG/ArchUnit-Examples (this has a simplified example build of the last release, no `@Category` or other special stuff, simple code, run it -> fails)
- #22 -> The new user guide is live now at http://archunit.org

Note: I've set a CNAME record for www.archunit.org (instead of archunit.org) + a redirect from `@` to www.archunit.org, since my domain provider doesn't allow ALIAS records, and I didn't want to link to a static IP address...

What do you think, are you missing something?

**Comment by gernotstarke** [comment]
whow, really great!!
That's what I initially hoped for - very well done!

Please allow for some (nitpicker) comments (if you like, I open issues in the new website repo...)

1. combining examples and docu-website imho throws a "cohesion violation exception". Imho better to create a distinct repo for the website, including userguide.
2. The site is missing an "about us" part, where you introduce the team (ahm, yourself + TNG),
that's imho interesting and quite common. 
3. The footer is missing a reference to TNG as supporter
4. In Germany website operators are imho required to add an "imprint", naming a responsible
person and contact. Having such a page does not hinder anyone, but might avoid the nasty "Abmahnungsproblem" sometimes heard of.
5. Add a twitter handle in the footer 
6. In arc42.org, we discuss of moving out of github-generated pages again, back to "ftp upload". Reason is that github does not allow https certs for custom domains.
7. I had the same concerns about linking to the static github IP addresses - but that proved to be a stable configuration for more than 12 month now...

Please note - usually I'm not that picky and obsessed with details... want me to create PR's for some of that?

keep up the great work - thanx and regards!

**Comment by codecholeric** [comment]
That's alright, I'm always open to improve things :wink: 

1. I guess you're right, since now you have to clone docs + examples, which doesn't make that much sense, maybe it's back to the good old gh-pages Branch then (SNAPSHOT in docs, release in gh-pages). I'll look into that, now that I'm on it anyway :wink: 
2. I'll look into that
3. dito
4. dito
5. Created a specific handle `@archtests` (everything else was unfortunately taken), I'll add that to the bottom and tweet news about ArchUnit from there from now on.
6. I've noticed that, too, at the moment I didn't see it as deal breaker, but I was surely disappointed, that I couldn't configure https. I think for now I'll keep it at http, if there is no strong reason against it. JGiven's page is hosted like that, too.
7. Yes, I guess the IPs are somewhat static enough, otherwise no one would do that :wink:, but still, redirecting from archunit.org to www.archunit.org doesn't seem too bad...

Thanks for the offer, but I guess those are all small things that I should take care of myself. I'll do it over the next couple of days...

**Comment by codecholeric** [comment]
Okay, check it now :wink:
Release docs are now hosted with ArchUnit on gh-pages, ArchUnit-Examples only contains the examples.
I've added "About", adjusted the footer, added an Imprint (I just took the same we had for our swacamp.org website) and added the new Twitter handle.
I've also added Gradle tasks to do all the updates automatically in the future, i.e. from a checked out release tag bring ArchUnit-Example and the gh-pages branch up to date...
